test/model-test.js

```diff
@@ -81,7 +81,7 @@ describe('AGOL Model', function () {
     })
 
     it('should call getItemMetadata to json', function (done) {
-      agol.getItem('host1', 'item1', {getMetadata: true}, function (err, json) {
```

Comment:
@dmfenton so you're no longer getting metadata at all? or just in a more on demand manner?